test/e2e/framework/framework.go

```diff
@@ -801,7 +801,7 @@ func setNextGCPMachineSize(current, family, subfamily, subfamilyflavor string, m
 			}
 			// For the N2 machine series, select between 0.5 GB and 8.0 GB per vCPU, inclusive.
 			// Note: the max is 864GB (with 80vCPUs we reach ~410GB).
```

Comment:
```suggestion
			// Note: the max is 864GB (with 80vCPUs we reach ~246GB).
```

lgtm, just one outdated comment

Reply:
Got rid of it. Thanks.